packages/components/src/components/mobile-dialog/mobile-dialog.jsx

```diff
@@ -123,11 +126,16 @@ const MobileDialog = props => {
     );
 };
 
+MobileDialog.defaultProps = {
```

Comment:
I believe we should avoid such implementation of default props, it's better to describe it inside func params

Reply:
Yes, remove it pls
https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/default_props/

Reply:
Let's remove this and add it in the props destructuring part.